odpf/optimus/runtime_service.proto

```diff
@@ -21,6 +21,7 @@ option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_swagger) = {
         description: "Optimus server";
     }
     schemes: HTTP;
+    hosts: "localhost:9100/api";
```

Comment:
@scortier @kushsharma What is the need of this?

Reply:
While using any api from optimus locally , 
1. The curl path is not correct ,so have to change it every time while using it.
2. The port is only to sync up with the optimus server , same as in `.optimus.yaml`.

Reply:
@scortier aren't both same binaries ? Can't we set the defaults in same place for both in the codebase itself?

Reply:
There are two ways we can sort this:
1. Adding `host` entry with the default port optimus server starts.
2. Parsing json file in optimus server, find out which port optimus is running, add json `host` field with the provided host, serialize it back and host it on some optimus route.

At the moment, optimus doesn't host swagger file on server endpoint but when we do host, I think 2nd point could be used to avoid polluting the proto file but until that is done I think 1st approach should be fine so that even if they do run this swagger file locally, swagger UI works?

Also it should be `127.0.0.1:9100/api`

Reply:
👍 
We should move `/api` to `base_path` field.